### PR TITLE
testnode: Filter ansible_mounts with regex

### PR DESCRIPTION
**Pull request by zmc**
This fixes an issue Seagate was hitting

### CODE REVIEW

**Comment by djgalloway** [comment]
Oooo I like this.  Can we combine all those options into one jinja2 regex?

File: roles/testnode/tasks/zap_disks.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@
   with_items: "{{ ansible_mounts }}"
   when:
     - item.mount != '/' and
-      item.mount != '/var' and
+      not item.mount.startswith('/var') and
```
Comment (djgalloway) [inline roles/testnode/tasks/zap_disks.yml:35]: Would `startswith('/var|/tmp|/root|/etc')` work?

Comment (zmc) [reply roles/testnode/tasks/zap_disks.yml:35]: I don't see why not, should I make that change?

Comment (djgalloway) [reply roles/testnode/tasks/zap_disks.yml:35]: Yeah!

Comment (zmc) [reply roles/testnode/tasks/zap_disks.yml:35]: Looks like I could replace all but the first condition (`!= /`) with a regex. Sound good?

**Review by djgalloway — APPROVED** [review]

**Pull request merged**